arches/app/templates/javascript.htm

```diff
@@ -494,9 +494,12 @@
                 'text': '{{ renderer.title }}',
                 'type': '{{ renderer.type }}',
                 'ext': '{{ renderer.ext }}',
+                'paths': '{{ renderer.paths }}',
```

Comment:
why is `paths` being added?

Reply:
good catch; this is cruft from an earlier thought about isolating renderers to specific paths.  Turns out - that's hard and not as useful.